src/model_service.cpp

```diff
@@ -198,6 +198,20 @@ Status GetModelStatusImpl::getAllModelsStatuses(std::map<std::string, tensorflow
         modelsStatusesTmp.insert({pipelineName, response});
     }
 
+    const std::vector<std::string>& mediapipePipelineNames = manager.getMediapipeFactory().getMediapipePipelinesNames();
+    for (auto const& mediapipePipelineName : mediapipePipelineNames) {
+        std::optional<int64_t> noValueModelVersion;
+        tensorflow::serving::GetModelStatusRequest request;
+        GetModelStatusImpl::createGrpcRequest(mediapipePipelineName, noValueModelVersion, &request);
+        tensorflow::serving::GetModelStatusResponse response;
+        auto status = GetModelStatusImpl::getModelStatus(&request, &response, manager, context);
+        if (status != StatusCode::OK) {
+            // Same situation like with models.
```

Comment:
Shouldn't we do something? 
Is it expected ?

Reply:
Expected.